apps/prairielearn/src/schemas/schemas/infoCourse.json

```diff
@@ -69,11 +74,12 @@
       "items": {
         "description": "A single assessment set description.",
         "type": "object",
+        "additionalProperties": false,
```

Comment:
Note to self, check if this and similar changes in this file will be breaking.

Reply:
This will indeed be breaking; this and similar changes in this file break 743 courses with ~1100 total errors.

Reply:
This has been resolved in the latest commits.